deploy/cloud-foundry/README.md

```diff
@@ -2,15 +2,42 @@
 
 ## Deployment Steps
 
-The quickest way to install Stratos is to deploy it as a Cloud Foundry application. To do so, clone the `stratos` repository, cd into the newly cloned repository and push to Cloud Foundry. This can be done with:
+The quickest way to install Stratos is to deploy it as a Cloud Foundry application. 
+
+You can do it in two ways:
+
+1. Deploy Stratos using the [`splatform/stratos`](https://hub.docker.com/r/splatform/stratos) docker image 
+
+    This can be done with command `cf push -f manifest-docker.yml`:
+    ```yaml
+    # manifest-docker.yml
```

Comment:
For ease of use this can be a new file (with same `manifest-docker.yml` title) committed to github